app/views/foreman_tasks/tasks/dashboard/_latest_tasks_in_error_warning.html.erb

```diff
@@ -0,0 +1,2 @@
+<h4><%=_("Latest Warning/Error Tasks ")%></h4>
```

Comment:
A link to the list of all tasks with warning or error could be useful here: basically to expand this limited view to the full one